doc-content/enterprise-only/springboot/openshift-springboot-proc.adoc

```diff
@@ -0,0 +1,167 @@
+   </container>
+ </containers>
+</kie-server-state>
+----
++
+. In the `business-application-kjar` project directory, edit the `pom.xml` file. Under the `<build>` tag, add the following lines:
++
+[source,xml]
+----
+  <resources>
+    <resource>
+      <directory>${basedir}</directory>
+      <includes>
+        <include>business-application-service.xml</include>
+      </includes>
+      <filtering>true</filtering>
+    </resource>
+  </resources>
+----
++
+. Outside the project directories, create an `ocp-image` directory with the following subdirectories:
++
+----
+ocp-image
+|--/root
+  |--/opt
+    |-- /m2
+      |--  /repo
+    |-- /spring-service
+----
++
+. In the `root/opt/m2` subdirectory, create a `settings.xml` file with the following content:
++
+[source,xml]
+----
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+http://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <localRepository>/opt/m2/repo</localRepository>
+  <offline>true</offline>
+</settings>
+----
++
+. Build the `business-application-service` project and copy the built service it into the `root/opt/spring-service` subdirectory, for example:
++
+----
+cd ../business-application-service
+mvn clean package
+mv target/business-application-service-1.0-SNAPSHOT.jar ../ocp-image/root/opt/spring-service/
+----
++
+. Build the `business-application-kjar` project and copy the built and deployed artifacts into the `root/opt/m2/repo` subdirectory, for example:
++
+----
+cd ../business-application-kjar/
+mvn clean install -P docker
+cp -R target/local-repository/maven/* ../ocp-image/root/opt/m2/repo/
+cp target/classes/business-application-service.xml ../ocp-image/root/opt/spring-service/
+----
++
+[NOTE]
+====
+This step relies on the `docker` POM configuration profile that is available in standard {PRODUCT} business applications available from http://start.jbpm.org.
+====
++
+. From the same built `business-application-kjar` project, copy the `business-application-service.xml` {KIE_SERVER} status file:
++
+----
+cp target/classes/business-application-service.xml ../ocp-image/root/opt/spring-service/
+----
++
+. In the `ocp-image` directory, create a `Dockerfile` file with the following content:
++
+----
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
+ENV M2_HOME=/opt/m2 GC_MAX_METASPACE_SIZE=512
+COPY root /
+EXPOSE 8090
+WORKDIR /opt/spring-service/
+CMD ["sh","-c", "java ${JAVA_OPTIONS} -Dorg.kie.server.controller=${KIESERVER_CONTROLLERS} -Dorg.kie.server.controller.user=${KIE_SERVER_CONTROLLER_USER} -Dorg.kie.server.controller.pwd=${KIE_SERVER_CONTROLLER_PWD} -Dorg.kie.server.mode=PRODUCTION -Dorg.kie.server.startup.strategy=LocalContainersStartupStrategy -Dkie.maven.settings.custom=/opt/m2/settings.xml -Dorg.guvnor.m2repo.dir=/opt/m2/repo -jar /opt/spring-service/business-application-service-1.0-SNAPSHOT.jar"]
```

Comment:
Another typical system property for spring boot app is the one which specifies the active config profile, i.e.:

-Dspring.profiles.active=prod